demo-simple/src/main/java/com/novoda/downloadmanager/demo/MainActivity.java

```diff
@@ -137,17 +131,23 @@ public void onReceived(List<DownloadBatchStatus> downloadBatchStatuses) {
         bindViews();
     }
 
-    VersionOneDatabaseCloner.CloneCallback cloneCallback = new VersionOneDatabaseCloner.CloneCallback() {
+    private final VersionOneDatabaseCloner.CloneCallback cloneCallback = new VersionOneDatabaseCloner.CloneCallback() {
```

Comment:
What's this callback for? Isn't the migrator callback enough?

Reply:
This is for some different functionality - the demo app has the ability to create a v1 database for testing purposes, this relates to that